src/master/restore.cc

```diff
@@ -835,7 +835,7 @@ int restore_line(const char* filename, uint64_t lv, const char* line) {
 				status = do_append(filename,lv,ts,ptr+6);
 			} else if (strncmp(ptr,"ACQUIRE",7)==0) {
 				status = do_acquire(filename,lv,ts,ptr+7);
-			} else if (strncmp(ptr,"AQUIRE",6)==0) {
```

Comment:
This suggestion modifies the intention of string comparison since size of string to compare has been hard coded. Should not be accepted.

Reply:
added pragma to ignore this line